db/schema.rb

```diff
@@ -28,11 +26,11 @@
 
   create_table "application_letters", force: :cascade do |t|
     t.string   "motivation"
-    t.integer  "user_id",                      null: false
-    t.integer  "event_id",                     null: false
-    t.datetime "created_at",                   null: false
-    t.datetime "updated_at",                   null: false
-    t.integer  "status",           default: 2, null: false
+    t.integer  "user_id",                               null: false
+    t.integer  "event_id",                              null: false
+    t.datetime "created_at",                            null: false
+    t.datetime "updated_at",                            null: false
+    t.integer  "status",                    default: 2, null: false
```

Comment:
no idea where all those changes come from, dropping and migrating didn't make them go away, so it's problem a problem on dev